Content.Client/State/GameScreen.cs

```diff
@@ -94,79 +95,49 @@ private void DisposePresenters()
             _constructionMenu?.Dispose();
         }
 
-
-        private void OnOocEnabledChanged(bool val)
+        internal static void FocusChat(ChatBox chat)
         {
-            _oocEnabled = val;
-
-            if (_adminManager.IsActive())
+            if (chat.UserInterfaceManager.KeyboardFocused != null)
             {
                 return;
             }
 
-            if(_gameChat is null)
-                return;
-
-            _gameChat.Input.PlaceHolder = Loc.GetString(_oocEnabled ? "Say something! [ for OOC" : "Say something!");
+            chat.Input.IgnoreNext = true;
+            chat.Input.GrabKeyboardFocus();
         }
-
-        private void OnAdminOocEnabledChanged(bool val)
+        internal static void FocusChannel(ChatBox chat, ChatChannel channel)
         {
-            _adminOocEnabled = val;
-
-            if (!_adminManager.IsActive())
+            if (chat.UserInterfaceManager.KeyboardFocused != null)
             {
                 return;
             }
 
-            if (_gameChat is null)
-                return;
-
-            _gameChat.Input.PlaceHolder = Loc.GetString(_adminOocEnabled ? "Say something! [ for OOC" : "Say something!");
-        }
-
-        private void OnAdminStatusUpdated()
-        {
-            if (_gameChat is null)
-                return;
-
-            _gameChat.Input.PlaceHolder = _adminManager.IsActive()
-                ? Loc.GetString(_adminOocEnabled ? "Say something! [ for OOC" : "Say something!")
-                : Loc.GetString(_oocEnabled ? "Say something! [ for OOC" : "Say something!");
+            chat.Input.IgnoreNext = true;
+            chat.SelectChannel(channel);
```

Comment:
Missing a `chat.Input.GrabKeyboardFocus();` if the keybinds are supposed to focus chat when pressed.